dashing/distribution.yaml

```diff
@@ -329,6 +329,14 @@ repositories:
       url: https://github.com/ros2/console_bridge_vendor.git
       version: dashing
     status: maintained
+  cyclonedds:
+    source:
+      test_commits: false
+      test_pull_requests: false
```

Comment:
What's the consensus here? I added them only because I saw them on other entries.

Reply:
Please reference those here since they should be removed too.

Reply:
> Please reference those here since they should be removed too.

I'm not sure what you mean.

Here is the check that fails unless these are set when specifying a commit: https://github.com/ros/rosdistro/blob/158a6037e658f8196e979969e9c41031a355a493/test/test_url_validity.py#L131-L133

Reply:
Oh, I didn't know about that unconditional check in Travis check in this repo. Then I guess it must stay.